src/xr.js

```diff
@@ -75,6 +75,31 @@ function promise(args, fn) {
   )(fn);
 }
 
+function getURI(opts) {
+	let uri = opts.url;
```

Comment:
It'd be nicer if the spacing conventions used in the current codebase were matched, rather than creating differences in the coding style... (that goes for the space in `if (` as well)